python/ray/data/aggregate.py

```diff
@@ -217,6 +217,24 @@ def finalize(a: List[float]):
         )
 
 
+class UniqueValues(_AggregateOnKeyBase):
+    """Defines unique value aggregation.
+
+    Computes a list of all (de-duped) values.
+    """
+
+    def __init__(self, on: Optional[KeyFn] = None):
+        self._set_key_fn(on)
+        on_fn = _to_on_fn(on)
+
+        super().__init__(
+            init=lambda k: [],
+            accumulate=lambda a, r: list(set(a) | {on_fn(r)}),
```

Comment:
Nit: Either use the `|` operator or `.union()`, but don't mix usage.
```suggestion
            accumulate=lambda a, r: list(set(a).union({on_fn(r)})),
```

Reply:
Dropping this since we got rid of this altogether!